python/langsmith/schemas.py

```diff
@@ -171,6 +171,22 @@ class Config:
         frozen = True
 
 
+DatasetTransformationType = Literal[
```

Comment:
Any thoughts on not making this a literal so it's easier to add new ones server side?

Reply:
I think since I union with str it at least won't be blocked if you try to say a new one